data/textile/export.py

```diff
@@ -154,12 +149,7 @@ def uuidOrSearch(txt):
 
     # export processes
     with open(PROCESSES, "w") as outfile:
-        outfile.write(
-            pprint.pformat(
-                json.dumps(list(processes.values()), indent=2, ensure_ascii=False),
-                width=100,
-            )
-        )
+        json.dump(list(processes.values()), outfile, indent=2, ensure_ascii=False)
         # Add a newline at the end of the file, to avoid creating a diff with editors adding a newline
         outfile.write("\n")
-    print(f"Exported {len(processes)} processes to {PROCESSES}")
+    print(f"Exported {len(processes)} processes to {PROCESSES}")
```

Comment:
hmm I thought we decided we actually always wanted a trailing newlines in all files including Python ones

Reply:
indeed, corrected